messages/shared.md

```diff
@@ -1,6 +1,6 @@
 # label.command-state
 
-Developer Preview
+Beta
```

Comment:
[FIX-AS-YOU-GO item] We don't want to forget to update this to say "Beta" instead of "Developer Preview" now.